simple-stack-example-kotlin-masterdetail-fragments/src/main/java/com/example/ktdagger/userdetail/UserDetailKey.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.ktdagger.userdetail
+
+import com.example.ktdagger.BaseKey
+import com.example.ktdagger.realmobjects.UserRO
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
+data class UserDetailKey(
+    val tag: String
+) : BaseKey() {
+
+    private lateinit var userRO: UserRO;
```

Comment:
If this field is not part of the primary constructor, then this field won't be detected by Parcelize. The constructors should be swapped.